package.json

```diff
@@ -16,7 +16,7 @@
     "build-sass": "rm -rf dist/public/style.css && sass --no-source-map src/assets/scss/application.scss dist/public/stylesheets/application.css --style compressed",
     "watch-sass": "sass --no-source-map src/assets/scss/application.scss dist/public/stylesheets/application.css --style compressed --watch",
     "copy-locales": "mkdir -p dist && copyfiles -u 1 src/**/*.njk dist/ src/locales/**/** dist/ ",
-    "copy-images": "copyfiles -u 3 src/assets/images/** dist/public/images",
+    "copy-images": "copyfiles -u 3 src/assets/images/** dist/public/images && copyfiles -u 5 node_modules/govuk-frontend/govuk/assets/images/** dist/public/images",
```

Comment:
Should copy the other way round so our choice overrides govuk-frontend assets if there's an overlap